linter.py

```diff
@@ -56,83 +54,11 @@ class Flake8(PythonLinter):
         '--max-line-length=': None,
         '--max-complexity=': -1,
         '--jobs=': '1',
-        '--show-code=': False
+        'show-code': False,
+        'executable': ''
```

Comment:
Shouldn't you also define a `executable` class attribute when you're removing `module`, or is this solely to circumvent https://github.com/SublimeLinter/SublimeLinter3/issues/455?

I would much rather not have to specify the flake8 executable path in my settings if SL is technically capable of locating it by itself.